src/main/java/com/iauto/eschool/config/security/RoleEnum.java

```diff
@@ -0,0 +1,72 @@
+package com.iauto.eschool.config.security;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public enum RoleEnum {
+	ADMIN(Set.of( PermissionEnum.CATEGORY_READ,
+			PermissionEnum.CATEGORY_WRITE,
```

Comment:
We can use import static Permission.*, then we can access enum element directly without mentioning enum class.